parquet-variant/src/main/java/org/apache/parquet/variant/VariantBuilder.java

```diff
@@ -508,7 +545,7 @@ public void endObject() {
    * Starts appending an array to this variant builder. The returned VariantArrayBuilder is used to
    * append values ot the array. startArray() must be called before endArray(). No append*() methods
    * can be called in between startArray() and endArray().
-   *
+   * <p>
```

Comment:
Unlike Scaladoc that behaves more like markdown, Javadoc requires `<p>` in order to start new paragraphs.